views/tenants/new.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Tenants</title>
+</head>
+<body>
+  <h1>Complete this Form to Add a Tenant:</h1>
+  <form method='post' action='/tenants/new'>
```

Comment:
should POST to `/tenants`. Because the method is POST, creating a new tenant is implied and is not necessary in the URL.